common/changes/office-ui-fabric-react/chf-2018-11-fileCard_expandTimeLog_2018-11-14-21-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "add a callback function to log hover card expand time",
```

Comment:
Please consider formatting the comment as

```json
"comment": "HoverCard: Add a callback function invoked when card expanded"
```

These change files are rolled-up into our release notes so it is good practice to clearly explain any public API changes. In this PR you've added a callback function which is invoked when a card is expanded. It does not log expand time.

Reply:
Good point. I think we are switching from "what was added" to "what to expect" as a consumer of the library in our language of the changelogs I think. 

HoverCard: A new callback called `onCardExpand` has been added that occurs after the expanded hover card is rendered